forte/processors/misc/simple_processors.py

```diff
@@ -58,3 +59,30 @@ def _process(self, input_pack: DataPack):
 
         if start < len(input_pack.text):
             input_pack.add_entry(Token(input_pack, start, len(input_pack.text)))
+
+
+class EntityMentionInserter(PackProcessor):
```

Comment:
Are you using this class for test case? Please don't implement it in the main src, implement this class in the test case folders.